bookmarks/images/templates/images/image/detail.html

```diff
@@ -12,15 +12,26 @@ <h1>{{ image.title }}</h1>
   <a href="{{ image.image.url }}">
     <img src="{% thumbnail image.image 300x0 %}" class="image-detail">
   </a>
-  {% with total_likes=image.users_like.count %}
+  {% with total_likes=image.users_like.count users_like=image.users_like.all %}

```

Comment:
```suggestion
  {% with total_likes=image.users_like.count users=image.users_like.all %}

```
Thats a list of users. The name miscommunicates the intent.